packages/common-common/src/permissions.ts

```diff
@@ -1,63 +1,123 @@
 export enum Action {
-  INVITE_MEMBERS = 0,
-  BAN_MEMBERS = 1,
-  MANAGE_COMMUNITY = 2,
-  ADMINISTRATOR = 3,
-  MANAGE_ROLES = 4,
-  MANAGE_WEBHOOKS = 5,
-  MANAGE_TOPICS = 6,
-  MANAGE_CHAT_CHANNELS = 7,
-  VIEW_COMMUNITY_INSIGHTS = 8,
-  MANAGE_INVITES = 9,
-  VIEW_TOPIC = 10,
-  VIEW_CHAT_CHANNELS = 11,
-  CREATE_THREAD = 12,
-  MANAGE_THREADS = 13,
-  CREATE_CHAT = 14,
-  CREATE_REACTION = 15,
-  CREATE_COMMENT = 16,
-  CREATE_POLL = 17,
-  VOTE_ON_POLLS = 18,
-  MANAGE_POLLS = 19,
+  CREATE_CHAT = 0,
+  VIEW_CHAT_CHANNELS = 1,
+  CREATE_REACTION = 2,
+  VIEW_REACTIONS = 3,
+  CREATE_COMMENT = 4,
+  VIEW_COMMENTS = 5,
+  EDIT_COMMENT = 6,
+  DELETE_COMMENT = 7,
+  CREATE_POLL = 8,
+  VIEW_POLLS = 9,
+  VOTE_ON_POLLS = 10,
+  CREATE_THREAD = 11,
+  VIEW_THREADS = 12,
+  EDIT_THREAD = 13,
+  DELETE_THREAD = 14,
 }
 
+const IMPLICIT_PERMISSIONS_BY_ACTION = new Map<number, Action[]>([
```

Comment:
Can this be displayed more cleanly, with line breaks, etc?

Reply:
We should also audit this list, to ensure we have the correct values.

Reply:
Auditing and reformatted via linter